docs/site/Accessing-http-request-response.md

```diff
@@ -0,0 +1,222 @@
+---
+lang: en
+title: 'Accessing http request and response objects'
```

Comment:
nitpick: it might be better to use `HTTP` instead of `http` throughout this doc?

```suggestion
title: 'Accessing HTTP request and response objects'
```